docs/fieldnorm/verification_validation.md

```diff
@@ -1,7 +1,70 @@
\ No newline at end of file
+> Verification and Validation
+> The verification and validation (V&V) activities should be targeted towards the criticality and impact to patient safety of the SaMD, as discussed in IMDRF/SaMD WG/N12.Typically, verification (providing assurance that the design and development activity at each development stage conforms to the requirements) and validation (providing reasonable confidence that the software meets its intended use/user needs and operational requirements) activities ensure that all elements from the SaMD design and development—including any changes made during maintenance/upgrades—have been implemented correctly and that objective evidence of this implementation is recorded.
+
+In short, Barry Boehm expressed the difference between the software verification and software validation as follows:
+
+> Verification: Are we building the product right?
+> Validation: Are we building the right product?
+
+Often these definitions do not match with the definitions used in the medical laboratory. However, in terms of software development and thus bioinformatics, following software verification and software validation definitions make more sense. Noteworthy, the ISO-15189 states that validation is needed for laboratory designed or developed methods and for validated methods subsequently modified. Both are true for software development.
+
+The IEC-62304 mentiones different aspects of verifation; i.e. software unit verification and software integration testing, and validation; i.e. software system test.
+
+## Software verification
+
+Testing whether the code is correctly developed can be done on different levels, for example through:
+
+- Code reviews
+- Unit testing
+- Integration testing
+
+## Software validation
+
+In order to support the repeatability of the validation, the following must be registered:
+
+- The person executing the test
+- The person evaluating the test
+- Date of the test
+- The software tested, including its version number
+- If applicable: configuration, hardware, testing tools
+- The test, including steps to take the test, expected results and acceptance criteria
+- The results, including samples used
+- The evaluation of the test
+- Conclusion of the test
+
+The person executing the test should be independent from the person who developed the software and wrote the validation. Software validation is preferably executed by an end user. For example, someone from the laboratory when the end result is mostly used by the laboratory.
```

Comment:
This seems quite tough for certain cases. Not all bioinformatic software development can be hands on used by end users as they are fully automatic or will be integrated into existing workflows. In these cases I could imagine that it makes more sense to have testing being performed by the developers while having the results validated by the end users.